chart/templates/mayastor/agents/core/agent-core-deployment.yaml

```diff
@@ -59,14 +59,16 @@ spec:
             - "--events-url=nats://{{ .Release.Name }}-nats:4222"{{ end }}
             {{- if not .Values.agents.ha.enabled }}
             - "--disable-ha"{{ end }}
+            - "--fmt-style={{ include "logFormat" . }}"
```

Comment:
you seem to have undone the macro calling?

Reply:
@tiagolobocastro , What do you mean?

Btw this is how it will be passed. tpl fn will invoke error if value is not json | pretty | compact

- "--ansi-colors=true"
- "--fmt-style=pretty"

Reply:
[17:15] Tiago Castro

I thought I had in the PR previously passing both json and color options using the macro

[17:16] Tiago Castro

        args:

           - "--fmt-style={{ include "logFormat" . }}"

           - "--ansi-colors={{ .Values.base.logging.color }}"

versus

        args:

           {{ include "logOptions" . }}

Reply:
This was the latest commit with macro call. its onlt for validating and passing fmt-style. ansi-colors is passed as usual assignment